Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1395,6 +1396,14 @@ theorem abs_sub_round_div_natCast_eq {m n : ℕ} :
   rw [abs_sub_round_eq_min, Nat.cast_min, ← min_div_div_right hn'.le,
     fract_div_natCast_eq_div_natCast_mod, Nat.cast_sub (m.mod_lt hn).le, sub_div, div_self hn'.ne']
 
+theorem round_half_bot (x : α) : x - 1 / 2 < round x := by
+  rw [round_eq x, show x - 1 / 2 = x + 1 / 2 - 1 by ring]
```

Comment:
```suggestion
  rw [round_eq x, show x - 1 / 2 = x + 1 / 2 - 1 by linarith]
```
letting linarith prove this avoids having to import `Tactic.Ring`

Reply:
FYI `linarith` uses `ring` under the hood, so your comment is actually to replace a tactic by a more expensive tactic.

Reply:
Are you sure? Importing `Mathlib.Tactic.Linarith` doesn't let me use the `ring` tactic

Reply:
See for yourself: https://github.com/leanprover-community/mathlib4/blob/8973ec00e9e76fc00ed67953e0ca960781f76e4c/Mathlib/Tactic/Linarith/Datatypes.lean#L7